### PR TITLE
core, api: store labels case-insensitive

### PR DESCRIPTION
**Pull request by Connor1996**
case-insensitive when comparing StoreLabel.key, the string store in StoreLabel is still case-sensitive

### CODE REVIEW

File: server/core/store.go [hunk 1]
```diff
@@ -193,8 +194,8 @@ func (s *StoreInfo) IsDisconnected() bool {
 // GetLabelValue returns a label's value (if exists).
 func (s *StoreInfo) GetLabelValue(key string) string {
 	for _, label := range s.GetLabels() {
-		if label.GetKey() == key {
-			return label.GetValue()
+		if strings.EqualFold(label.GetKey(), key) {
+			return strings.ToLower(label.GetValue())
```
Comment (disksing) [inline server/core/store.go]: I prefer returning `label.GetValue()` here, use `strings.EqualFold` outside to compare.

**Comment by siddontang** [comment]
PTAL @huachaohuang

**Pull request merged**